subsys/bluetooth/controller/ll_sw/lll_filter.h

```diff
@@ -28,53 +28,73 @@ struct lll_filter {
 
 /* Filter Accept List peer list */
 struct lll_fal {
-	uint8_t      taken:1;
-	uint8_t      id_addr_type:1;
-	uint8_t      rl_idx;
+	uint8_t   taken:1;
+	uint8_t   id_addr_type:1;
+	uint8_t   rl_idx;
 	bt_addr_t id_addr;
 };
 
-struct lll_resolvelist {
-	uint8_t      taken:1;
-	uint8_t      rpas_ready:1;
-	uint8_t      pirk:1;
-	uint8_t      lirk:1;
-	uint8_t      dev:1;
-	uint8_t      fal:1;
+/* Periodic Advertising List */
+struct lll_pa_list {
+	bt_addr_t id_addr;
+	uint8_t   taken:1;
+	uint8_t   id_addr_type:1;
+	uint8_t   sid;
+
+#if defined(CONFIG_BT_CTLR_PRIVACY)
+	uint8_t   rl_idx;
+#endif /* CONFIG_BT_CTLR_PRIVACY */
+};
 
-	uint8_t      id_addr_type:1;
+/* Resolve list */
+struct lll_resolve_list {
+	uint8_t   taken:1;
+	uint8_t   rpas_ready:1;
+	uint8_t   pirk:1;
+	uint8_t   lirk:1;
+	uint8_t   dev:1;
+	uint8_t   fal:1;
+
+	uint8_t   id_addr_type:1;
 	bt_addr_t id_addr;
 
-	uint8_t      local_irk[16];
-	uint8_t      pirk_idx;
+	uint8_t   local_irk[16];
```

Comment:
I think I saw a IRK_SIZE #define somewhere in a C file that possibly could be moved to a header file and used here as well.